### PR TITLE
ACMS-1721: Update Cypress to latest and include in ACMS CI

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**
Fixes #ACMS-1721

**Proposed changes**
Update Cypress to latest and include in ACMS CI

**Alternatives considered**
NA

**Testing steps**
- Follow from ticket

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: tests/cypress/cypress/e2e/specsACMS/acms.content.spec.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    })
+
+    //TC-79
+    context("Add Content - Click and verify", () => {
+        it("Click on Add content on Content tab from sub-admin tool bar, ", () => {
+            content.clickAndVerify()
+        })
+    })
+
+    //TC-80
```
Comment (apathak18) [inline tests/cypress/cypress/e2e/specsACMS/acms.content.spec.js:31]: What is this TC signifies?

File: tests/cypress/cypress/e2e/specsACMS/acms.content.spec.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    //Article
+    context("Article - Click and verify", () => {
+        it("Mouse hover on Content tab and navigate to Article link from sub-admin tool bar", () => {
+            article.clickAndVerify()
+        })
+    })
+
+    //Verify and extend required modules
```
Comment (apathak18) [inline tests/cypress/cypress/e2e/specsACMS/acms.content.spec.js:39]: Comment line should have 1 space before the statement and ended with the fullstop.

File: tests/cypress/cypress/pages/Article.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    }
+
+    //Article Save button
+    get articleSave() {
+        return cy.get("#edit-submit")
+    }
+
+    //get created article title
+    get createdArticleTitle() {
+        return cy.get("body > div.dialog-off-canvas-main-canvas > div.coh-container.coh-style-focusable-content.coh-ce-6f78460f > div > article > div.coh-container.coh-style-padding-top-medium.coh-style-padding-bottom-large.coh-container-boxed > div > div > div.coh-column.coh-style-padding-bottom-small.coh-visible-ps.coh-col-ps-12.coh-visible-sm.coh-col-sm-10.coh-visible-xl.coh-col-xl-8 > h1")
```
Comment (apathak18) [inline tests/cypress/cypress/pages/Article.js:50]: These element selector `body > div.dialog-off-canvas-main-canvas > div.coh-container.coh-style-focusable-content.coh-ce-6f78460f > div > article > div.coh-container.coh-style-padding-top-medium.coh-style-padding-bottom-large.coh-container-boxed > div > div > div.coh-column.coh-style-padding-bottom-small.coh-visible-ps.coh-col-ps-12.coh-visible-sm.coh-col-sm-10.coh-visible-xl.coh-col-xl-8`

In createdArticleType, createdArticleAuthor and createdArticleContent

Could we try to make one variable and pass that variable where it is required.

File: tests/cypress/cypress/pages/Event.js [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+    }
+    //save the event button
+    get eventSave() {
+        return cy.get("#edit-submit")
+    }
+
+    //Validation
+    //Fetch created event title
+    get fetchCreatedTitle() {
+        return cy.get("body > div.dialog-off-canvas-main-canvas > div.coh-container.coh-style-focusable-content.coh-ce-6f78460f > div > article > div.coh-container.coh-style-padding-top-medium.coh-style-padding-bottom-large.coh-container-boxed > div:nth-child(1) > div > div > h1")
```
Comment (apathak18) [inline tests/cypress/cypress/pages/Event.js:83]: Common elements: body > div.dialog-off-canvas-main-canvas > div.coh-container.coh-style-focusable-content.coh-ce-6f78460f > div > article > div.coh-container.coh-style-padding-top-medium.coh-style-padding-bottom-large.coh-container-boxed > div:nth-child(1) > div > div

In fetchCreatedTitle and fetchPlaceEvent

File: tests/cypress/cypress/pages/MenuTabs.js [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+        })
+        cy.url().should('eq', testData.$people_url)
+
+        this.placesMenu.click({
+            force: true
+        })
+        cy.url().should('eq', testData.$places_url)
+
+    }
+    spell_validations_primary_menu() {
```
Comment (apathak18) [inline tests/cypress/cypress/pages/MenuTabs.js:106]: Add a small comment to describe what this custom function do

File: tests/cypress/cypress/pages/TourPage.js [hunk 6]
```diff
@@ -0,0 +1,394 @@
...
+        }).should('have.text', 'Site Studio core')
+        this.textBoxApiKey.should('be.visible')
+        this.textBoxAgencyKey.should('be.visible')
+        this.saveButtonSSC.should('be.visible')
+        this.ignoreButtonSSC.should('be.visible')
+        this.advancedDescriptionSSC.should('be.visible')
+        this.advancedDescriptionIconSSC.should('be.visible')
+        this.siteStudioCore.click()
+
+        //TODO - enable the module
```
Comment (chandan-singh7929) [inline tests/cypress/cypress/pages/TourPage.js:282]: Which module we have to enable here as part of TODO

**Review by chandan-singh7929 — APPROVED** [review]
LGTM, there are some minor suggestions.

**Review by deepakmishra2 — APPROVED** [review]
LGTM

File: tests/cypress/cypress/pages/MenuTabs.js (hunk 5 above)
Comment (rajeshreeputra) [reply tests/cypress/cypress/pages/MenuTabs.js:107]: done

File: tests/cypress/cypress/e2e/specsACMS/acms.content.spec.js (hunk 2 above)
Comment (rajeshreeputra) [reply tests/cypress/cypress/e2e/specsACMS/acms.content.spec.js]: done

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**